ggml/src/ggml-cuda/ggml-cuda.cu

```diff
@@ -521,7 +521,8 @@ struct ggml_cuda_pool_vmm : public ggml_cuda_pool {
 };
 #endif // defined(GGML_USE_VMM)
 
-std::unique_ptr<ggml_cuda_pool> ggml_backend_cuda_context::new_pool_for_device(int device) {
+std::unique_ptr<ggml_cuda_pool> ggml_backend_cuda_context::new_pool_for_device(int                  device,
+                                                                               [[maybe_unused]] int stream_no) {
```

Comment:
Nit: I'm personally not a fan of adding unused function arguments (it's not only maybe_unused, but simply not used at all)